src/esp/nav/PathFinder.cpp

```diff
@@ -229,6 +231,8 @@ struct PathFinder::Impl {
 
   std::pair<vec3f, vec3f> bounds() const { return bounds_; };
 
+  std::vector<std::vector<bool>> getTopDownView(const float res);
```

Comment:
Let's give `res` a more intelligible name.  Seems like it is being used as `metersPerPixel`?